src/components/NavBar/NavBar.jsx

```diff
@@ -19,11 +18,11 @@ const NavBar = () => {
             <i className="fa fa-caret-down">Articles</i>
           </button>
           <div className='dropdown-content'>
-          <Link to="AllArticles">All Articles</Link>
-            <Link to="Article 1">Article 1</Link>
+          <Link to="articles">All Articles</Link>
+            <Link to="SearchArticle">Search</Link>
+            <Link to="/Article 1">Article 1</Link>
```

Comment:
What if there were 500 articles? A single link to view all articles is a better choice here 👍